### ISSUE TITLE
[Docs]: aws-iam-authenticator is required?

### ISSUE THREAD

**Issue by dweebo** [kubefirst/kubefirst#1161]
### What type of suggestions?

Missing Information

### What is the issue?

When running `kubefirst cluster create` I got an error
```
Error: error installing argo-cd: install argo-cd
```

The logs showed
```
LOG: 2023/01/15 13:06:22.991388 /home/runner/work/kubefirst/kubefirst/pkg/shell.go:26: ERR: Error: Kubernetes cluster unreachable: Get "https://C33269CB40ABFA2641813EC7DA685A03.gr7.us-east-1.eks.amazonaws.com/version?timeout=32s": getting credentials: exec: executable aws-iam-authenticator not found
```

Once I installed `aws-iam-authenticator` I was able to get past this step.  I didn't see this prerequisite documented anywhere.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 6za** [comment]
Thank you for the feedback, we will update the docs to add this requirements to AWS based installations.

**Comment by 6za** [comment]
PR #1161 confirms the need of the tool to be installed. Further work maybe be done to automate it installation for some OS flavors supported. For example, `brew` receipt for Macos.  Such as PR: #1162.

**Comment by 6za** [comment]
Removing it from release, as documentation is been updated to reflect it, further work can be delivered post-1.11 release, not a blocker.

**Comment by converge** [comment]
hi @dweebo , thanks for the report, we were able to reproduce and fix the issue. It will be released in the next release that should come in the next days.

Now `brew install` have aws iam authenticator as a dependency that will be downloaded during the brew install step. We also updated our docs to reflect the change. (the documentation will be updated at the time of the new release)

**Comment by converge** [comment]
@dweebo fyi; we released 1.11.0 yesterday, and this was included, thanks again!

**Comment by dweebo** [comment]
Awesome thanks!
